src/lib/Behat/BrowserContext/NavigationContext.php

```diff
@@ -135,6 +135,14 @@ public function iGoToUserNotifications()
         $this->upperMenu->chooseFromUserDropdown('View Notifications');
     }
 
+    /**
+     * @Given I log out of backoffice
```

Comment:
Looks like our doc uses "back office": https://doc.ibexa.co/en/latest/extending/extending_back_office/#back-office
```suggestion
     * @Given I log out of back office
```